README.adoc

```diff
@@ -83,9 +83,41 @@ that the correct docker image is used. An example snippet:
 |helmArtifactId |`"helm"` |The helm binary `artifactId`
 |helmVersion |None |The helm binary `version`
 |helm.skip |`false` |If true, execution will be skipped entirely
+|strictLint |`false` |If true, linting fails on warnings (see: <<goal-lint>>)
+|valuesFile | None | values file that should be used for goals <<goal-lint>>, <<goal-template>>
+|outputFile | target/test-classes/helm.yaml | output file for <<goal-template,template goal>>
 
 |===
 
+== Goals
+
+[#goal-package]
+=== Package
+
+goal packages a chart directory into a chart archive using the https://github.com/helm/helm/blob/master/docs/helm/helm_package.md[helm package] command.
```

Comment:
Could you start every sentence in the descriptions with a capital letter? Thanks.